pyproject.toml

```diff
@@ -38,6 +38,9 @@ nbsphinx = {version = "^0.7.1", optional = true}
 IPython = {version = "^7.18.1", optional = true}
 torch = "^1.8.0"
 skorch = "^0.9.0"
+# The poetry2nix build of matplotlib fails for newer versions. See
+# https://github.com/nix-community/poetry2nix/issues/280#issuecomment-815064853
+matplotlib = "<3.4"
```

Comment:
Hm, it would be inconvenient, if we have to restrict dependencies just because poetry2nix has problems building it. The dependency specification should only specify a set of dependency ranges for which we know the package works.